tests/e2e/conftest.py

```diff
@@ -54,13 +54,15 @@ def init_project(project_name: str, libs_path: str):
 
 @pytest.fixture
 def protostar():
-    def _protostar(args: List[str]) -> str:
+    def _protostar(args: List[str], check=True) -> str:
```

Comment:
1. It's not obvious what is checked.
2. Default booleans should be set to False.

`ignore_exit_code=False` maybe?